store/index.go

```diff
@@ -0,0 +1,324 @@
+// Copyright 2019 Tim Shannon. All rights reserved.
```

Comment:
Left this in here... It actually is barely any code from badgerhold now, but it was inspired by that work, so seemed appropriate to leave here.